upload/upload.go

```diff
@@ -126,23 +130,34 @@ func uploadFiles(files, outFiles []string, targetDir string, config *helpers.Con
 		}
 		_ = f.Close()
 	}
+	s3Endpoint := config.HostBase
+	if !strings.HasPrefix(s3Endpoint, "http") {
+		s3Endpoint = fmt.Sprintf("http://%s", config.HostBase)
+		if !config.UseHTTPS {
+			s3Endpoint = fmt.Sprintf("https://%s", config.HostBase)
+		}
+	}
```

Comment:
```suggestion
```

This is handled in the LoadConfig function.
Although it is not perfect since it makes a bunch of assumptions on how the entry will look like.